AB_Hands_On/AB_Hands_On.ipynb

```diff
@@ -0,0 +1,6511 @@
+{
```

Comment:
<blockquote>you already went through some parts of what's called the Bayesian workflow. Let's recap and talk about that.</blockquote><p> </p><p>This already is perfect intro for the next (proposed) section where you go through everything again in one shot and show how to actually run an ab comparison with the tools we've seen</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='74'/>